tools/vscode_workspace/merge.sh

```diff
@@ -0,0 +1,11 @@
+WORKSPACE=../../engine.code-workspace
+cleaned_temp_file=$(mktemp)
+json5 "$WORKSPACE" -s 2 -o "$cleaned_temp_file"
```

Comment:
what is json5 and yq? I'm using zsh.

Reply:
They are both tools that are available with homebrew.  I've updated the README.md to mention them.  Since they aren't required for building or testing the project I suspect they are fine as implicit dependencies.  Originally I started creating a dart tool that did this.  This is preferable to developing our own bespoke tools.